services/management/ia/templates_service.go

```diff
@@ -338,18 +481,84 @@ func (svc *TemplatesService) ListTemplates(ctx context.Context, req *iav1beta1.L
 }
 
 // CreateTemplate creates a new template.
-func (svc *TemplatesService) CreateTemplate(ctx context.Context, req *iav1beta1.CreateTemplateRequest) (*iav1beta1.CreateTemplateResponse, error) {
-	return nil, status.Errorf(codes.Unimplemented, "method CreateTemplate not implemented")
+func (s *TemplatesService) CreateTemplate(ctx context.Context, req *iav1beta1.CreateTemplateRequest) (*iav1beta1.CreateTemplateResponse, error) {
+	pParams := &alert.ParseParams{
```

Comment:
`pParams` -> `parseParams`

Reply:
Well, I don't know, the scope of that variable is a few lines. Does it require a more verbose name?

Reply:
nah, its always better to have self-explantory names, but thats a nit